frontend/test/metabase/visualizations/components/Visualization-table.unit.spec.js

```diff
@@ -0,0 +1,53 @@
+jest.mock("metabase/components/ExplicitSize");
+
+import React from "react";
+import { render, cleanup } from "@testing-library/react";
```

Comment:
Are we still using going to use this for unit tests?

Reply:
I copy pastad that from pie chart tests, but no probably not. Need to finish up #10478